packages/babel-plugin-transform-typescript/src/enum.ts

```diff
@@ -320,15 +322,11 @@ function computeConstantValue(
       }
 
       if (seen.has(path.node)) return;
+      seen.add(path.node);
 
-      const bindingInitPath = path.resolve(); // It only resolves constant bindings
-      if (bindingInitPath) {
```

Comment:
`path.resolve()` always returns something, so this check wasn't actually guarding anything.